daemon/src/collab_settlement/protocol.rs

```diff
@@ -30,7 +30,7 @@ const DECISION_TIMEOUT: Duration = Duration::from_secs(30);
 
 pub const SETTLEMENT_MSG_TIMEOUT: Duration = Duration::from_secs(120);
 
-#[tracing::instrument]
+#[tracing::instrument(skip(endpoint))]
```

Comment:
```suggestion
#[tracing::instrument(skip(endpoint), skip(collab_settlement_tx)]
```

if the collab settlement tx does not match, it will get printed anyway; in a happy case, we don't need it :)

Reply:
I've amended the commit and pushed!